js/kucoin.js

```diff
@@ -1895,10 +1895,13 @@ module.exports = class kucoin extends Exchange {
         if (cancelExist) {
             status = 'canceled';
         }
+        if (status === undefined) {
+            status = 'closed';
+        }
```

Comment:
@carlosmiei what's the reason to parse "nothing" as "closed"? 

this makes every `createOrder` call automatically return the status "closed" - which is wrong  unless it's a market order (and even then, it's information that ccxt invented, not  something the exchange provided).

Reply:
@carlosmiei kindly reminder for the above question.

Reply:
> @carlosmiei what's the reason to parse "nothing" as "closed"?
> 
> this makes every `createOrder` call automatically return the status "closed" - which is wrong unless it's a market order (and even then, it's information that ccxt invented, not something the exchange provided).

@carlosmiei This is true, I going to remove this addition in a PR

Reply:
@xmatthias @samgermain sorry guys I missed this comment